### PR TITLE
Move up_args after the up command.

### PR DESCRIPTION
**Pull request by ghost**
## Summary
Fixes issue #984 where up_args are being placed before up, and need to go after.

## Additional Context
Since 'compose' became an argument, the insert statement just need to be incremented by one to represent the new position.  The other insert seems fine where it is.

## Related Issues (if any)
#984

- [x] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: lib/puppet/provider/docker_compose/ruby.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ def get_image(service_name, compose_services)
 
   def create
     Puppet.info("Running compose project #{name}")
-    args = ['compose', compose_files, '-p', name, 'up', '-d', '--remove-orphans'].insert(3, resource[:options]).insert(5, resource[:up_args]).compact
+    args = ['compose', compose_files, '-p', name, 'up', '-d', '--remove-orphans'].insert(3, resource[:options]).insert(6, resource[:up_args]).compact
```
Comment (smortex) [inline lib/puppet/provider/docker_compose/ruby.rb:77]: The usage of `#insert` make it a bit hard to follow.  I think we could benefit from re-writing this as something like:

```ruby
args = ['compose', compose_files, '-p', name, resource[:options], 'up', resource[:up_args], '-d', '--remove-orphans'].flatten.compact
```
(assuming the index of the first insert is wrong because I don't think it is correct to insert `resource[:options]` between `-p` and `name`).

Also consistently using long options helps for readability: `-p` => `--project-name`, `-d` => `--detach`

Comment (ncstate-daniel) [reply lib/puppet/provider/docker_compose/ruby.rb:77]: If y'all want to do it a different way that's certainly cool with me.  =)  I needed this fixed asap and submitted my fix without modifying the way things are done too much.

Comment (ncstate-daniel) [reply lib/puppet/provider/docker_compose/ruby.rb:77]: (fwiw i agree -- i don't entirely understand why it was done the way it was)

**Comment by jst-cyr** [comment]
This PR was never approved or updated by the original contributor and now has conflicts that must be resolved. The original contributor is now a deleted user. Going to close this for now and if it is needed a new PR can be opened.
